contributor_ladder.md

```diff
@@ -0,0 +1,188 @@
+
+### Community Participant
+
+Description: A Community Participant participates in the community and contributes their time, thoughts, etc. Community participants are usually users who have stopped being anonymous and started being active in project discussions.
+
+* Responsibilities:
+    * Must follow the Konveyor CoC
+* How users can get involved with the community:
+        * Participating in community discussions
+        * Helping other users
+        * Submitting bug reports
+        * Commenting on issues
+        * Trying out new releases
+        * Attending community events
+
+
+### Project Contributor
+
+Description: A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+* Responsibilities include:
+    * Following the Konveyor CoC
+    * Following the project contributing guide
+* Requirements (one or several of the below):
+    * Reports and sometimes resolves issues
+    * Occasionally submits PRs
+    * Contributes to the documentation
+    * Regularly shows up at meetings, takes notes
+    * Answers questions from other community members
+    * Submits feedback on issues and PRs
+    * Tests releases and patches and submits reviews
+    * Runs or helps run events
+    * Promotes the project in public
+    * Helps run some part of the project infrastructure
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become a Project Member
+
+
+### Project Member
+
+Description: A Project Member is an established contributor who regularly participates the project. Project Members have privileges in both project repositories and elections, and as such are expected to act in the interests of the whole project.
+
+An Project Member must meet the responsibilites and has the requirements of a Contributor, plus:
+
+* Responsibilities include:
+    * Continues to contribute regularly, as demonstrated by having at least [TODO: Number] contributions a year, as demonstrated by [TODO: contributor metrics source].
+
+* Requirements:
+    * Must have successful contribitons to the project, including at least one of the following:
+        * [TODO: Number] accepted PRs,
+        * Reviewed [TODO: Number] PRs,
+        * Resolved and closed [TODO: Number] Issues,
+        * Become responsible for a key project management area,
+        * Or some equivalent combination or contribution
+    * Must have been contributing for at least [TODO: Number] months
+    * Must have two sponsors who are also Project Members
+    * Must enable 2FA on their GitHub account
+    * [TODO: other technical requirements]
+
+
+* Privileges:
+    * May be assigned Issues and Reviews
```

Comment:
This implies that issues or reviews may not be assigned to lower roles-- if someone is being mentored  over a Good First Issue, then this seems like an awkward restriction.

It also falls apart when dealing with nascent projects-- kind of a bootstrapping problem.

Reply:
Kevin: Github does not permit us to assign issues and reviews to someone who is not an organization member.  That's why this is in there.